variousauthors/heart-projector-online
# Kay fireroom branch split up

This branch is meant to include all the work from #16  but split up into the format proposed by #18 

This branch is intended to replace #16 

**QA Steps:**
 - @somnambulant  could you check out this branch, start it up, and go through the rooms and make sure I didn't clobber anything. I might have missed a line, or forgotten something. Just make sure none of your work has disappeared...
 - @4xisblack and @negaspace could you start up the app and make sure that your work hasn't been changed in subtle or weird ways by this merge... that includes the heart projector "oldspace" rooms.
